_posts/2023-06-20-trino-fest-2023-recap.md

```diff
@@ -0,0 +1,122 @@
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>AWS Athena
+  (Trino) in the cybersecurity space](https://youtu.be/WCuJaW7zC8k) presented by
+  Anas Shakra from [Artic Wolf](https://arcticwolf.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Skip rocks
+  and files: Turbocharge Trino queries with Hudi’s multi-modal indexing
+  subsystem](https://youtu.be/IiDOmAEOXUM) presented by Nadine Farah and  Sagar
+  Sumit from [OneHouse](https://www.onehouse.ai/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Redis &
+  Trino - Real-time indexed SQL queries (new
+  connector)](https://youtu.be/JjBtZ26IHYk) presented by Allen Terleto and
+  Julien Ruaux from [Redis](https://redis.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Let it SNOW
+  for Trino](https://youtu.be/kmpO_yM8OAs) presented by Erik Anderson from
+  [Bloomberg](https://www.bloomberg.com/company/values/tech-at-bloomberg/open-source/projects/)
+  and Yu Teng from [ForePaaS](https://www.forepaas.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>DuneSQL, a
+  query engine for blockchain data](https://youtu.be/sCJncarnGdU) presented by
+  Miguel Filipe and Jonas Irgens Kylling from [Dune](https://dune.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Data Mesh
+  implementation using Hive views](https://youtu.be/ZgcVtPFkKHM) presented by
+  Alejandro Rojas from [Comcast](https://comcast.github.io/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Inspecting
+  Trino on ice](https://youtu.be/PSGuAMVc6-w) presented by Kevin Liu from
+  [Stripe](https://stripe.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Trino
+  optimization with distributed caching on Data
+  Lake](https://youtu.be/oK1A5U1WzFc) presented by Hope Wang and Beinan Wang
+  from [Alluxio](https://www.alluxio.io/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Starburst
+  Galaxy: A romance of many architectures](https://youtu.be/K3AlAWB-Gmg)
+  presented by Benjamin Jeter from [Datto](https://www.datto.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>FugueSQL,
+  Interoperable Python and Trino for interactive
+  workloads](https://youtu.be/aKhI1Phfn-o) presented by [Kevin
+  Kho](https://www.linkedin.com/in/kvnkho/).
+
+## Next up
+
+With this first recap we are sharing all the video recordings with you all. Next
+up are individual blog post for each session that include further useful
+material:
+
+* Presentation file to look at your own pace
+* Video recording
+* Interesting and fun quotes from the speakers and audience
+* Notes and impressions from the audience and event hosts
+* Questions and answer during the event
+* Links to further documentation, tutorials, and other resources
+
+Putting all this together will take us a while, so keep an eye out on our
```

Comment:
```suggestion
We'll be rolling out recap posts for a few talks each week, so keep an eye out on our
```
I think "it's gonna take a while" makes it sound like we're incapable of doing work faster. We _are_ capable, we're just choosing to spread the content out to drive more engagement/views on LinkedIn and send a larger volume of traffic to the website over time, rather than in one big burst.